geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ha/HAGIIDUnitTest.java

```diff
@@ -109,18 +111,21 @@ public final void postSetUp() throws Exception {
 
   @Test
   public void testGIIRegionQueue() {
-    client0.invoke(() -> HAGIIDUnitTest.createEntries());
-    client0.invoke(() -> HAGIIDUnitTest.registerInterestList());
-    server0.invoke(() -> HAGIIDUnitTest.put());
-
-    server0.invoke(() -> HAGIIDUnitTest.tombstonegc());
-
-    client0.invoke(() -> HAGIIDUnitTest.verifyEntries());
-    server1.invoke(HAGIIDUnitTest.class, "createServer2Cache", new Object[] {new Integer(PORT2)});
-    Wait.pause(6000);
-    server0.invoke(() -> HAGIIDUnitTest.stopServer());
-    // pause(10000);
-    client0.invoke(() -> HAGIIDUnitTest.verifyEntriesAfterGiiViaListener());
+    try (IgnoredException ignoredException =
```

Comment:
Since this is in scope for the entire test, you could remove the try-with-resource syntax and just use:
```
addIgnoredException(ConnectException.class);
```
I also like to use `import static org.apache.geode.test.dunit.IgnoredException.addIgnoredException`

Reply:
All IgnoredExceptions are automatically added during DUnit tearDown if you use either a DistributedTestCase type class or any of the Distributed rules.